matchesWhereQuery.js

```diff
@@ -3,45 +3,73 @@ var _ = require( 'underscore' );
 module.exports = function( object, whereQuery ) {
 	var whereQueryKeys = _.keys( whereQuery );
 	for( var i = 0; i < whereQueryKeys.length; i++ ) {
-		var thisKey = whereQueryKeys[ i ];
-		var thisKeyQuery = whereQuery[ thisKey ];
-		var thisObjectValue = object[ thisKey ];
+		var attribute = whereQueryKeys[ i ];
```

Comment:
I think we can use another name for this `var`. I personally prefer `key` over `attribute`. What did you think @dgbeck .
Thx!

Reply:
seems pretty similar to me, no preference, my head spins for all of these names!

Reply:
Oks, sounds good. 👍

Reply:
Talked to Javi and changed it to attributeName. :)